### PR TITLE
Adding failed property to request.

### PR DESCRIPTION
**Pull request by cdamian**

### CODE REVIEW

File: src/ParsedRequest.php [hunk 1]
```diff
@@ -135,4 +135,15 @@ public function getParams();
      * @return ParsedRequest
      */
     public function setParams(ParameterBag $params);
+
+    /**
+     * @return bool
+     */
+    public function isFailed() : bool;
```
Comment (Kingson-de) [inline src/ParsedRequest.php]: shouldn't it be hasFailed ?

Comment (cdamian) [reply src/ParsedRequest.php]: Thought of it to be in tune with isEnabled() :D

Comment (hendrikmaus) [reply src/ParsedRequest.php]: I would also pledge for `hasFailed`

**Comment by hendrikmaus** [comment]
@cdamian please resolve the merge conflict

File: src/ParsedRequest.php [hunk 2]
```diff
@@ -194,7 +194,7 @@ public function setRepetitionConfig(Repetition $config) : ParsedRequest;
     /**
      * @return bool
      */
-    public function isFailed() : bool;
+    public function hasFailed() : bool;
```
Comment (hendrikmaus) [inline src/ParsedRequest.php:197]: perfect!
FYI @Kingson-de

**Comment by cdamian** [comment]
@hendrikmaus Done.

**Pull request merged**